app/javascript/App/bundles/common/components/CoinListItem.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import PercentageChange from './PercentageChange'
 import WatchButton from './WatchButton'
 
 export default (props) => {
   const { coin, user, onClick } = props
   let klass = 'pa-default b--b flex items-center pointer'
-  //if (isActiveEntity({ type: 'coin', id: coin.get('id') })) klass += ' bg-foam'
+
+  if (props.isSelected) {
+    klass += ' bg-foam'
```

Comment:
Is this something that `classNames` can help with?

Reply:
Yes, it definitely needs to be rewritten with `classnames` function to make it more readable. Thanks!